docs/dev-guide/plugin-api.md

```diff
@@ -1 +1,97 @@
 # Plugin API
+
+## getCwd
+
+- **Usage**:
+Returns a current working directory
+
+## resolve
+
+- **Arguments**
+  - `{string} _path` - relative path from project root
```

Comment:
Do `_` need ?

Reply:
Yes, because the source code for Plugin API is:
```
const path = require('path')
...

resolve (_path) {
  return path.resolve(this.service.context, _path)
}
```
So we need `_` to avoid the conflict with the global `path`

Reply:
I think we can omit the `_` in the context of the docs.

Reply:
Ok, fixed